Dockerfile

```diff
@@ -0,0 +1,16 @@
+# base image
+FROM node:12.2.0-alpine
```

Comment:
Do we want to use node 12?

Reply:
Discussed with team - I think it's good unless we find a reason not to.